hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlAvroTest.java

```diff
@@ -407,6 +379,40 @@ public void test_topLevelFieldExtraction() {
         );
     }
 
+    @Test
+    public void test_explicitKeyAndValueSerializers() {
+        String name = createRandomTopic();
+        sqlService.execute("CREATE MAPPING " + name + " ("
+                + "key_name VARCHAR EXTERNAL NAME \"__key.name\""
+                + ", value_name VARCHAR EXTERNAL NAME \"this.name\""
+                + ") TYPE " + KafkaSqlConnector.TYPE_NAME + ' '
+                + "OPTIONS ( "
+                + '"' + OPTION_KEY_FORMAT + "\" '" + AVRO_FORMAT + '\''
+                + ", \"" + OPTION_VALUE_FORMAT + "\" '" + AVRO_FORMAT + '\''
+                + ", \"key.serializer\" '" + KafkaAvroSerializer.class.getCanonicalName() + '\''
```

Comment:
This test would actually pass even if the option got replaced, because we use the default value. We can use some missing class and check that it fails not finding it.

Again in `SqlJsonTest`

Reply:
I'm checking that existing options are not replaced in `PropertiesResolverTest`